server/repository/src/mock/vaccination.rs

```diff
@@ -16,6 +16,7 @@ pub fn mock_vaccination_a() -> VaccinationRow {
         vaccine_course_dose_id: mock_vaccine_course_a_dose_a().id,
         encounter_id: mock_immunisation_encounter_a().id,
         given: true,
+        patient_link_id: mock_patient().id,
```

Comment:
Tests were failing after I added name_link_id to the vaccination changelogs - have removed this now, per above, but keeping these changes so as to have more correct mock data